src/commands/deployWorkspaceProject/deployWorkspaceProjectSettings.ts

```diff
@@ -39,3 +42,25 @@ export async function setDeployWorkspaceProjectSettings(rootFolder: WorkspaceFol
         await settingUtils.updateWorkspaceSetting(`${deployWorkspaceProjectPrefix}.${key}`, settings[key], settingsPath, ConfigurationTarget.WorkspaceFolder);
     }
 }
+
+export function displayDeployWorkspaceProjectSettingsOutput(settings: DeployWorkspaceProjectSettings): void {
+    if (hasAllDeployWorkspaceProjectSettings(settings)) {
+        // Skip, more detailed logs will come when we confirm whether or not the resources were found
+    } else if (hasAtLeastOneDeployWorkspaceProjectSetting(settings)) {
+        ext.outputChannel.appendLog(localize('resourceSettingsIncomplete', 'Scanned and found incomplete containe app resource settings at "{0}".', relativeSettingsFilePath));
```

Comment:
Scanned reminds me of this lol

<img src="https://media2.giphy.com/media/tyttpGSdJWowdv47eCI/giphy.gif"/>

```suggestion
        ext.outputChannel.appendLog(localize('resourceSettingsIncomplete', 'Found incomplete container app workspace settings at "{0}".', relativeSettingsFilePath));
```